src/nspawn/nspawn-settings.c

```diff
@@ -1019,3 +1021,53 @@ int config_parse_bind_user(
 
         return 0;
 }
+
+int parse_bind_user_shell(const char *s, char **ret) {
+        int r;
+
+        if (is_path(s))
+                *ret = strdup(s);
```

Comment:
Coding style: please update `*ret` only when the function succeeds. So, please introduce a temporal variable, and assign `ret` at the end of the function.

Reply:
Got it, good call, addressed in b173edc87dd3c01c6cb2ca9bb77092e160377501.